cmd/kubeadm/app/cmd/init.go

```diff
@@ -65,22 +65,27 @@ func NewCmdInit(out io.Writer, s *kubeadmapi.KubeadmConfig) *cobra.Command {
 		&s.InitFlags.API.ExternalDNSNames, "api-external-dns-names", []string{},
 		`(optional) The DNS names to advertise, in case you have configured them yourself.`,
 	)
-
 	cmd.PersistentFlags().IPNetVar(
 		&s.InitFlags.Services.CIDR, "service-cidr", *kubeadmapi.DefaultServicesCIDR,
-		`(optional) use alternative range of IP address for service VIPs, e.g. "10.16.0.0/12"`,
+		`(optional) Use alterantive range of IP address for service VIPs, defaults to `+
+			kubeadmapi.DefaultServicesCIDRString,
+	)
+	cmd.PersistentFlags().IPNetVar(
+		&s.InitFlags.PodNetwork.CIDR, "cluster-cidr", net.IPNet{},
+		`(optional) Use alternative range of IP addresses for the pod overlay network VIPs, e.g. 10.3.0.0/16.`+
```

Comment:
Also, strictly speaking it wouldn't always mean that these IPs are used for overlay networks, e.g. in the where routes are setup in the cloud cloud provider.